src/components/Breadcrumb/Breadcrumb.tsx

```diff
@@ -21,13 +18,11 @@ export type BreadcrumbComponentCSSClasses = ComponentCSSClasses<
 export type BreadcrumbComponentTemplates = Required<BreadcrumbTemplates>;
 
 export type BreadcrumbProps = {
-  items: BreadcrumbItem[];
+  items: BreadcrumbConnectorParamsItem[];
   cssClasses: BreadcrumbComponentCSSClasses;
-  templateProps: {
-    templates: BreadcrumbComponentTemplates;
-  };
-  createURL(value: string | undefined): string;
-  refine(value: string | undefined): void;
+  templateProps: PreparedTemplateProps<BreadcrumbComponentTemplates>;
+  createURL(value?: string | null): string;
```

Comment:
Not sure why we need to handle`null` here. I don't know in which case is this value `null` ?

Reply:
https://github.com/algolia/instantsearch.js/blob/c57d7bbdd9451299b3af48ce6f894b012c24b9d5/src/connectors/breadcrumb/connectBreadcrumb.ts#L270

this is what makes it null, and I don't see how to avoid it (last item?)

Reply:
or right! that's when the value has been selected! 👍  thanks for clarifying.